src/components/Button.js

```diff
@@ -1,16 +1,29 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-const Button = ({ value }) => (
-  <div>{ value }</div>
-);
+const Button = ({ value, color, wide }) => {
+  const css = color;
+  if (wide) css.flex = '2';
```

Comment:
Optional: There is a small misalignment with the `0` button, you know how those TSEs can be really picky  😂.

![image](https://user-images.githubusercontent.com/2121481/78143362-c6eeb000-73f3-11ea-9789-8f7f7539123e.png)

Maybe you can try setting the flex-basis to 50%, something like this in your case, but you need to test it:

`if (wide) css.flex = '0 0 50%';`

Reply:
Thanks for that eagle eye! 🦅 , you are right is a small misalignment, now it looks better with the flex properties you suggested. 💯